### PR TITLE
[TAS-114] ✨ Add upload only mode

### PR DESCRIPTION
**Pull request by williamchong**
image
image

### CODE REVIEW

File: components/IscnRegisterForm.vue [hunk 1]
```diff
@@ -308,16 +334,35 @@
...
+                  'text-[8px]',
+                  'underline',
+                  'mt-[4px]',
+                  'mb-[-12px]',
+                ]"
+                href="#"
+                @click.prevent="showUploadOnly = true"
+              >
+                {{ $t('IscnRegisterForm.button.uploadOnly') }}
+              </a>
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:364]: I believe this button should be moved to the left, as the main action should always remain on the right-most side

Also, this `<a/>` could be replaced by `<Button preset="outline" />` to ensure consistent styling

File: components/IscnRegisterForm.vue [hunk 2]
```diff
@@ -308,16 +334,35 @@
...
-            <Button
-              v-else
-              :text="$t('IscnRegisterForm.button.register')"
-              type="submit"
-              preset="secondary"
+            <div v-else
+              :class="[
+                'flex',
+                'flex-col',
+                'items-end',
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:341]:
```suggestion
                'items-end',
                'gap-[12px]',
```

File: components/IscnRegisterForm.vue [hunk 3]
```diff
@@ -1020,6 +1068,33 @@ export default class IscnRegisterForm extends Vue {
     return undefined
   }
 
+  async onUploadOnly(): Promise<void> {
+    logTrackerEvent(this, 'ISCNCreate', 'ClickSubmit', '', 1);
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:1072]: Should we distinguish this from the `ClickSubmit` event in `onSubmit()`?

File: components/FileUploadedInfo.vue [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      >
+        <a
+          :href="`https://arweave.net/${arweaveId}`"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {{ arweaveId }}
+        </a>
+      </FormField>
+      <Divider class="mb-[24px]" />
```
Comment (nwingt) [inline components/FileUploadedInfo.vue:49]:
```suggestion
      <Divider v-if="$slots.footer" class="mb-[24px]" />
```

File: components/FileUploadedInfo.vue [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      <slot name="footer" />
```
Comment (nwingt) [inline components/FileUploadedInfo.vue:50]: Seems this is not used

File: components/FileUploadedInfo.vue [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      :has-padding="false"
+    >
+      <!-- header -->
+      <IscnFormHeader :step="step" :total-step="4"/>
+      <!-- guide title -->
+      <Label
+        :text="$t('FileUploaded.guide.title')"
+        class="text-medium-gray mt-[12px] mb-[28px]"
+      />
+      <slot name="card-footer" />
```
Comment (nwingt) [inline components/FileUploadedInfo.vue:17]: Seems this is not used

File: components/IscnRegisterForm.vue (hunk 1 above)
Comment (williamchong) [reply components/IscnRegisterForm.vue]: > I believe this button should be moved to the left, as the main action should always remain on the right-most side
> 
> Also, this `<a/>` could be replaced by `<Button preset="outline" />` to ensure consistent styling

Totally missed there are 2 parts of review..
image

Does this look ok?

**Pull request merged**